### ISSUE TITLE
the criterion in the mmwave_PC_hpe_tutorial.ipynb

### ISSUE THREAD

**Issue by yikai-lu** [FanJunqiao/mmDiff#8]
Appreacitate for you great work. 
In the mmwave_PC_hpe_tutorial.ipynb file, the criterion for model P4Transformer in dataset mmbody is crossentropy, which leads to a size bug. I wonder whether it is due to some particular intention or just a mistake.

**Comment by FanJunqiao** [comment]
Thank you for the use and for reporting the bug. The cross-entropy loss should be a bug, and the correct one should be MSE loss for regression. I will fix the tutorial.
